arangod/Cluster/ClusterInfo.cpp

```diff
@@ -2646,9 +2718,10 @@ int ClusterInfo::ensureIndexCoordinatorInner(std::string const& databaseName,
           // The mutex in the condition variable protects the access to
           // *errMsg:
           CONDITION_LOCKER(locker, agencyCallback->_cv);
-          errorMsg = *errMsg;
-          return tmpRes;
+
+          return Result(tmpRes,*errMsg);
```

Comment:
```suggestion
          return Result(tmpRes, *errMsg);
```
Do we still need `errMsg` here, or can it be removed?

Reply:
that's the string that gets populated in the asynchronous task lines 2571, 2587, its value gets returned as part of the function result